docs/source/_json/search.json

```diff
@@ -19,5 +25,5 @@ content-type: application/json
       "title": "Test Folder"
     }
   ], 
-  "items_count": 2
+  "total_items": 2
```

Comment:
@lukasgraf I'm wondering if it would make sense to rename "total_items" to "items_total". This would make "items_total" show up right after "items". Does not make any difference for the consumer app but for a developer browsing the API.